### DISCUSSION TITLE
website: add copy button to code blocks

### DISCUSSION THREAD

**Discussion by matrv** [ghostty-org/ghostty#3146] [Feature Requests, Ideas] [answered]
It would be great to have a copy button in the code blocks in the docs, similar to how it works on github:

![Image](https://github.com/user-attachments/assets/b258d159-03d0-4605-8338-d9450f2780ef)
![Image](https://github.com/user-attachments/assets/b62af0e7-1104-4142-9198-b697b8b8225b)

**Comment by mitchellh** [comment, marked as answer]
Approved, please open an issue on the website repo. I agree.

**Comment by pluiedev** [comment]
Resolved in ghostty-org/website#140
